application/src/main/java/org/opentripplanner/standalone/config/routerconfig/TransitRoutingConfig.java

```diff
@@ -112,7 +112,7 @@ public TransitRoutingConfig(
 no extra threads are started and the search is done in one thread.
 """
         )
-        .asInt(dft.searchThreadPoolSize());
+        .asInt(0);
```

Comment:
Why change this?

Reply:
The default does not exist any more. Raptor is not creating the threadPool - it is passed in, so there should not be a parameter on the RaptorApi for threadPoolSize. So dagger not inject thread-pool-size into `RaptorEnvironmentFactory`. So there is not natural place for the default value. I guess I could make it a constant in this class or in `RaptorEnvironmentFactory`.

Reply:
Thanks, that makes sense.